### PR TITLE
feat: add generate conda forge package meta script

### PR DESCRIPTION
**Pull request by floralatin**

### CODE REVIEW

**Comment by floralatin** [comment]
And please review the [commit](https://github.com/jina-ai/staged-recipes/commit/a45302df8e5adeb6e0f20f2f8819c24023ac1d3a)

**Review by nomagick — APPROVED** [review]
LGTM if the yaml needs to be preserved in git.

**Comment by floralatin** [comment]
> LGTM if the yaml needs to be preserved in git.

Yaml is to be preserved in git. 
And  [conda-forge/staged-recipes](https://github.com/conda-forge/staged-recipes) will auto trace the version of `jina-hubble-sdk` after conda-forge merge this`[ pr](https://github.com/conda-forge/staged-recipes/pull/20927)`

File: conda/meta.yaml
Comment (mapleeit) [inline conda/meta.yaml:2]: Is this version correct?

File: conda/meta.yaml
Comment (mapleeit) [inline conda/meta.yaml:10]: Shall we convert this fixed sha256 to a template variable?

File: conda/meta.yaml
Comment (mapleeit) [inline conda/meta.yaml:40]:
```suggestion
  home: https://github.com/jina-ai/jina-hubble-sdk/
  summary: SDK for Hubble API at Jina AI.
  dev_url: https://github.com/jina-ai/jina-hubble-sdk/
```

File: scripts/generate-conda-recipe.py
Comment (mapleeit) [inline scripts/generate-conda-recipe.py:12]: Is it necessary to have this info twice both here and in `meta.yml`?

File: conda/meta.yaml
Comment (mapleeit) [inline conda/meta.yaml:27]: Is this mandantory because we already have `requirements.txt`, maintaining them in two places is hard

**Review by mapleeit — CHANGES_REQUESTED** [review]

File: conda/meta.yaml
Comment (floralatin) [reply conda/meta.yaml:10]: > Shall we convert this fixed sha256 to a template variable?
No, it is  automatically created by running` python generate-conda-recipe.py`

File: conda/meta.yaml
Comment (floralatin) [reply conda/meta.yaml:40]: Maybe we should convert `hubble-client-python` to `jina-hubble-sdk`
 the following places in the file `setup.py `

[_project_urls ](https://github.com/jina-ai/jina-hubble-sdk/blob/main/setup.py#:~:text=%5D-,_project_urls,-%3D%20%7B)
[_download_url](https://github.com/jina-ai/jina-hubble-sdk/blob/main/setup.py#:~:text=hubble%2Dclient%2Dpython/%27-,_download_url,-%3D%20%27https%3A//github)

[_url](https://github.com/jina-ai/jina-hubble-sdk/blob/main/setup.py#:~:text=)-,_url,-%3D%20%27https%3A//github)

File: conda/meta.yaml
Comment (floralatin) [reply conda/meta.yaml:27]: This is not mandatory. We just recorded the information of the first version of meta.yaml, we don't need to keep track of it.
And [conda-forge/staged-recipes](https://github.com/conda-forge/staged-recipes) will auto trace the version of jina-hubble-sdk after conda-forge merge this https://github.com/conda-forge/staged-recipes/pull/20927

File: scripts/generate-conda-recipe.py
Comment (floralatin) [reply scripts/generate-conda-recipe.py:12]: This info  `maintainers = 'mapleeit nomagick delgermurun Andrei997 floralatin` in meta.yml,
 It is our configuration and the result of running `python generate-conda-recipe`.

**Review by mapleeit — APPROVED** [review]

**Pull request merged**
